dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -826,7 +836,9 @@ private String getFromWhereClause( EventSearchParams params, SqlHelper hlp,
             + "inner join program p on p.programid = pi.programid "
             + "inner join programstage ps on ps.programstageid = psi.programstageid "
             + "inner join categoryoptioncombo coc on coc.categoryoptioncomboid = psi.attributeoptioncomboid "
-            + "inner join organisationunit ou on psi.organisationunitid = ou.organisationunitid ";
+            + "inner join organisationunit ou on psi.organisationunitid = ou.organisationunitid "
+            + "left join users auc on (psi.assigneduserid=auc.userid) "
```

Comment:
spaces between "=" at least . maybe for "()" as well?

Reply:
This format was used consistently across the same file. Including your commit here https://github.com/dhis2/dhis2-core/blame/09ec208e2e9b0d0d8a80133c20db8d304d75cc56/dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java#L630  Should I still change it?

Reply:
no, then it's fine :)